Samples/VSTO-shared-code-migration/README.md

```diff
@@ -0,0 +1,169 @@
+
+UI code and document interaction code will need to be rewritten for the Office web add-in. However, business logic code can be shared between the VSTO add-in and Office web add-in.
+
+## Prepare the solution to run
+
+You'll need to isntall the required NuGet packages, and add test certificates for the solution to build and run correctly.
+
+1. Start Visual Studio 2019 and open the **/completed/Cell-Analyzer.sln** solution.
+2. In **Solution Explorer**, right-click **Solution 'Cell-Analyzer'**, and choose **Restore NuGet Packages**.
+3. In **Solution Explorer**, right-click the **Cell-Analyzer** project, and choose **Properties**.
+4. Choose the **Signing** category in the properties.
+5. Choose **Sign the ClickOnce manifests**, and then chose **Create Test Certificate**.
+6. In the **Create Test Certificate** dialog, enter and confirm a password. Then choose **OK**.
+
+## Part 1: Share code in a class library
+
+To share the business logic code, you create a class library. Then refactor the business logic and relocate it into the shared library. The advantage is that you will not need to rewrite the code in JavaScript for the Office web add-in. Later the Office web add-in will access the shared library through REST API calls.
+
+### Explore the shared class library
+
+1. Start Visual Studio 2019 and open the **/completed/Cell-Analyzer.sln** solution.
+2. Expand the **CellAnalyzerSharedLibrary** project in the solution explorer and open the **CellOperations.cs** file.
+
+The method **GetUnicodeFromText** is the shared function between both the VSTO add-in and Office web add-in.
+
+### Run the VSTO add-in with the shared class library
+
+1. Start Visual Studio 2019 and open the **/completed/Cell-Analyzer.sln** solution.
+2. In **Solution Explorer** right-click the **Cell-Analyzer** project and choose **Set as Startup Project**.
+3. From the **Debug** menu, choose **Start Debugging**.
+
+> **Note:** In the completed folder, the Cell Analyzer add-in is already configured to use the shared class library.
+
+## Part 2: Create a REST API wrapper for shared code
+
+The Office web add-in will need to access the shared library. To do this we'll provide a REST API that it can access. We can do this by creating an ASP.NET CORE application that references the class library.
+
+### Explore the REST API project
+
+1. Start Visual Studio 2019 and open the **/completed/Cell-Analyzer.sln** solution.
+2. Expand the **CellAnalyzerRESTAPI** project in the solution explorer and open the **/Controllers/AnalyzeUnicodeController.cs** file.
+
+The method **AnalyzeUnicode** is a wrapper that calls into the shared class library to perform the actual operation.
+
+### Run the REST API project
+
+1. Start Visual Studio 2019 and open the **/completed/Cell-Analyzer.sln** solution.
+2. In **Solution Explorer** right-click the **CellAnalyzerRESTAPI** project and choose **Set as Startup Project**.
+3. From the **Debug** menu, choose **Start Debugging**.
```

Comment:
I get an error message at this step.
![image](https://user-images.githubusercontent.com/38896772/71195087-12fd5680-2242-11ea-9c33-2a3def68b56d.png)

Reply:
The sample was built with .NET core 2.2. If you run the VS installer you can ensure you have the latest version. I'll update the readme to indicate the version the sample was built with.

Reply:
@davidchesnut Sounds good. However, is there a reason this is built using .NET core 2.2 instead of 3.1?